.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Description
+
+<!-- Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+- :bug: Fixes #(issue)
+- :rocket: Feature #(issue)
+
+## Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+- [ ] :bug: Bug fix (non-breaking change which fixes an issue)
```

Comment:
If the issue creation is enforced, this is bmjot mandatory but today the issue creation is not mandatory and not sure it can be until we are moving to Github Project or be plugged to it :S